src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_CropHarvestor.java

```diff
@@ -630,12 +632,14 @@ public boolean displaysStackSize() {
     public void saveNBTData(NBTTagCompound aNBT) {
         super.saveNBTData(aNBT);
         aNBT.setBoolean("mModeAlternative", this.mModeAlternative);
+        aNBT.setBoolean("mHarvestEnabled", this.mHarvestEnabled);
     }
 
     @Override
     public void loadNBTData(NBTTagCompound aNBT) {
         super.loadNBTData(aNBT);
         this.mModeAlternative = aNBT.getBoolean("mModeAlternative");
+        this.mHarvestEnabled = aNBT.getBoolean("mHarvestEnabled");
```

Comment:
when an existing harvester loads its nbt it will be set to false. I would recommend to invert the boolean or add a check if the tag exists.